source/extensions/common/wasm/wavm/wavm.cc

```diff
@@ -585,6 +620,36 @@ template void getFunctionWavm<uint32_t, uint32_t, uint32_t, uint32_t, uint32_t,
     std::function<uint32_t(Context*, uint32_t, uint32_t, uint32_t, uint32_t, uint32_t, uint32_t,
                            uint32_t, uint32_t, uint32_t, uint32_t)>*);
 
+template <typename T> T getValue(IR::Value) {}
```

Comment:
Is this necessary? SFINAE

Reply:
Yes, but it will fail to compile if is actually used because it doesn't return anything.